docs/arch/adr-005-betterproto.md

```diff
@@ -28,8 +28,24 @@ You can use the google protobuf package to generate Python "classes", using the
 * Additional benefits listed: [betterproto](https://github.com/danielgtaylor/python-betterproto)
 
 
+## Revisited
```

Comment:
Nice! Thanks for reviewing/rewriting. Let's put a date here (and up top)? I know it's available from `git blame`, but just for quick reference

Reply:
Dated.